### DISCUSSION TITLE
Missing CONFIG_IP_MROUTE_MULTIPLE_TABLES in ubuntu kernel (IPv6 is present!)

### DISCUSSION THREAD

**Discussion by choppsv1** [community/community#126638] [Actions]
### Select Topic Area

Product Feedback

### Body

The ubuntu kernel is missing the following config `CONFIG_IP_MROUTE_MULTIPLE_TABLES`. Interestingly it is enabled for IPv6, just not IPv4:

```
grepping MULTIPLE_TABLES
CONFIG_IP_MULTIPLE_TABLES=y
# CONFIG_IP_MROUTE_MULTIPLE_TABLES is not set
CONFIG_IPV6_MULTIPLE_TABLES=y
CONFIG_IPV6_MROUTE_MULTIPLE_TABLES=y
```

Could this option please be re-enabled in the github runner ubuntu kernel?

**Comment by choppsv1** [comment]
BTW, this is required to support multicast routing in VRFs.

**Comment by grandmas-favorite** [comment]
Hi @choppsv1!

We appreciate your feedback and your valuable input. We want to be transparent about our current priorities. You can gain insights into the projects we’re currently focusing on for the upcoming year by checking our public roadmap. While smaller fixes may not always be explicitly listed, they often get addressed when related work is in progress.

While we do review all incoming feedback unfortunately due to the volume of feedback, we may not be able to respond to each one individually at this time. However, we strongly encourage you to continue providing your valuable input as it greatly helps us prioritize and improve GitHub.

Your contributions are instrumental in making GitHub better for all users!
